### PR TITLE
Portfolio desktop PR

### PR DESCRIPTION
**Pull request by til2to**
Create Desktop version of the portfolio
Built with CSS and HTML

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,138 +9,336 @@
...
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </section>
       </article>
-  
-      <article class="tags">
+
+      <article class="skills">
+        <h3>Skills</h3>
```
Comment (roshan-bajgain) [inline index.html:71]: - [ ] I would suggest you, please change the color of `h3` tag which is: Language, Framework, and skills, and change the color of the verticle line as well. See the image below for reference:
### Your design
his
### Figma design
figma

File: index.html [hunk 2]
```diff
@@ -9,138 +9,336 @@
...
-          <li>html</li>
-          <li>javascript</li>
-          <li>Ruby</li>
+          <li><a href="#"><img src="one.png" alt=""></a></li>
+          <li><a href="#"><img src="Frame.png" alt=""></a></li>
+          <li><a href="#"><img src="three.png" alt=""></a></li>
+          <li><a href="#"><img src="four.png" alt=""></a></li>
+          <li><a href="#"><img src="five.png" alt=""></a></li>
         </ul>
       </article>
```
Comment (roshan-bajgain) [inline index.html:98]: - [ ] I would suggest you, please have the image bigger than the text content on the left. See the image for reference.
- [ ] Kindly make changes to all the sections.
### Your design
your
### Figma design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @til2to, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] [Optional] I would suggest you please wrap all the images in one folder for better management of the project.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @til2to , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**